launchlib/config_test.go

```diff
@@ -299,6 +300,7 @@ jvmOpts:
   - jvmOpt1
   - jvmOpt2
 dangerousDisableContainerSupport: true
+heapPercentage: 75
```

Comment:
should we update the README, or is this something you don't want to make broadly visible right now?

Reply:
I've updated the README to include this - I expect people to use it. I think this will be strictly better than manually overriding `MaxRAMPercentage` and `InitialRAMPercentage`